chapters/en/slides/module3/module3_06.md

```diff
@@ -0,0 +1,103 @@
+---
+type: slides
+title: 'module3_06'
+---
+
+# Pulling changes from a remote repository
+
+---
+# Pulling changes from the remote GitHub repository to synchronize your local repository
+
+<center>
+
+<img src='/module3/vc-pull.png' width="55%" alt="404 image"/>
+
+<center/>
+
+Notes: To obtain the new changes from the remote repository on GitHub, you will need to pull those changes to your own local repository. By pulling changes, you synchronize your local repository to what is present on GitHub. Additionally, until you pull changes from the remote repository, you will not be able to push any more changes yourself (though you will still be able to work and make commits in your own local repository)
+
+
+---
+
+#  Pulling changes from GitHub using JupyterLab 
+
+
+<center>
+
+<img src='/module3/vc-pull-jupyter.png' width="70%" alt="404 image"/>
+
+<center/>
+
+
+Notes: **1. The GitHub interface indicates the name of the last person to push a commit to the remote repository, a preview of the associated commit message, the unique commit identifier, and how long ago the commit was snapshotted.**
+
+---
+
+#  Pulling changes from GitHub using JupyterLab 
+
+
+<center>
+
+<img src='/module3/vc-pull-jupyter-1.png' width="70%" alt="404 image"/>
```

Comment:
I think we can show the terminal side by side here, it is relatively straighforward and not much text in the transcript.

I think we need to make all the images in this slide deck a bit bigger